pygaggle/run/evaluate_passage_ranker.py

```diff
@@ -40,6 +42,8 @@ class PassageRankingEvaluationOptions(BaseModel):
     index_dir: Path
     method: str
     model: str
+    duo_model: str
+    top: int
```

Comment:
rename this maybe? top sounds weird, go with mono_hits? Any other suggestions?

Reply:
mono_hits sounds good, updated